axum/src/routing/method_filter.rs

```diff
@@ -9,6 +9,24 @@ use std::{
 pub struct MethodFilter(u16);
 
 impl MethodFilter {
+    /// Match `CONNECT` requests.
+    ///
+    /// This is useful for implementing HTTP/2’s [extended CONNECT method],
```

Comment:
Weird nitpick, but I think it's pretty uncommon to use that unicode `’` in source code, even comments? I wouldn't even know how to type it 😅

Reply:
Hah, that’s force of habit from me 😅. Fixed now.